test/models/Call-test.ts

```diff
@@ -954,60 +904,19 @@ describe("ElementCall", () => {
             call.off(CallEvent.Layout, onLayout);
         });
 
-        it("ends the call immediately if we're the last participant to leave", async () => {
+        it("ends the call immediately if the session ended", async () => {
```

Comment:
Similar story here:
 - do dont mess around with state events anymore and instead trust the tests for the `MatrixRTCSessionManager` and only test if mocked changes in the session manager update/destroy the `Call`